### PR TITLE
Implement ParamDef copy

### PR DESCRIPTION
**Pull request by oSumAtrIX**

### CODE REVIEW

File: Confuser.Core/Helpers/InjectHelper.cs [hunk 1]
```diff
@@ -99,6 +99,9 @@ public static class InjectHelper {
 
 			newMethodDef.Signature = ctx.Importer.Import(methodDef.Signature);
 			newMethodDef.Parameters.UpdateParameterTypes();
+			
+            		foreach (var paramDef in methodDef.ParamDefs)
+               			newMethodDef.ParamDefs.Add(new ParamDefUser(paramDef.Name, paramDef.Sequence));
```
Comment (mkaring) [inline Confuser.Core/Helpers/InjectHelper.cs:104]: Please watch out for the indention. Only a single tab to indent the loop.
In addition the attributes of the Parameters should be copied as well, if we start with that.
```suggestion
               		newMethodDef.ParamDefs.Add(new ParamDefUser(paramDef.Name, paramDef.Sequence, paramDef.Attributes));
```

**Review by mkaring — CHANGES_REQUESTED** [review]
Where does this oversight cause an error? As far as I am aware, none of the runtime code that is injected requires these to be present.

File: Confuser.Core/Helpers/InjectHelper.cs (hunk 1 above)
Comment (oSumAtrIX) [reply Confuser.Core/Helpers/InjectHelper.cs:104]: ParamDefs are not required per se and no error appears at runtime ,but they should be included for the sake of completeness

File: Confuser.Core/Helpers/InjectHelper.cs [hunk 2]
```diff
@@ -99,64 +99,80 @@ public static class InjectHelper {
 
 			newMethodDef.Signature = ctx.Importer.Import(methodDef.Signature);
 			newMethodDef.Parameters.UpdateParameterTypes();
+			
+			foreach (var paramDef in methodDef.ParamDefs)
+               		newMethodDef.ParamDefs.Add(new ParamDefUser(paramDef.Name, paramDef.Sequence, paramDef.Attributes));
```
Comment (ElektroKill) [inline Confuser.Core/Helpers/InjectHelper.cs:104]: Mixed indent style! (tabs and spaces in the same line)

**Review by ElektroKill — CHANGES_REQUESTED** [review]

File: Confuser.Core/Helpers/InjectHelper.cs (hunk 2 above)
Comment (oSumAtrIX) [reply Confuser.Core/Helpers/InjectHelper.cs:104]:
```suggestion
				newMethodDef.ParamDefs.Add(new ParamDefUser(paramDef.Name, paramDef.Sequence, paramDef.Attributes));
```

**Review by oSumAtrIX — COMMENTED** [review]
Fix idention style

**Comment by mkaring** [comment]
Nice try fixing that complex method warning. :wink: The method is still too long for Codefactor to like it.

I noticed that the bracket style of your code doesn't line up with the rest of the code. Opening brackets are usually on the same line. For that not to happen, I got a `.editorconfig` setup in the project. I'm curious why that one isn't working for you.

**Review by mkaring — APPROVED** [review]

**Pull request merged**